backend/Entities/User.cs

```diff
@@ -0,0 +1,22 @@
+using System.Diagnostics.CodeAnalysis;
+
+namespace CbtBackend.Entities;
+
+public class User {
+    public int Id { get; set; }
+
+    [NotNull]
+    public string? Email { get; set; }
```

Comment:
You can use `public string Email { get; set; } = default!` which is more idiomatic and does not introduce nullability

Reply:
Same goes to all other instance of `[NotNull]` usage

Reply:
i was hoping that `[NotNull]` would prompt the deserialization to treat this property as required, but apparently it doesn't

Reply:
changed in https://github.com/shilangyu/SE2-CBT/pull/124/commits/1a7b23681c5e67a356b24fd05ed2376bfb60aefa